doc/user/interop.md

```diff
@@ -307,7 +362,8 @@ an integer, or anything else
 
 `object.new(*args)` sends `NEW`
 
-`object.respond_to?` calls `Truffle::Interop.respond_to?(object, message)`
+`object.respond_to?` calls `Truffle::Interop.respond_to?(object, message)`,
+which supports `to_a`, `to_ary`, `new` and returns `false` for everything else.
```

Comment:
"`to_a`, `to_ary`, `new` and returns `false`" -> "`to_a`, `to_ary`, and `new`, returning `false`"?

Reply:
Fixed.